plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -189,8 +211,14 @@ export function WriteItForMeButtonContainer() {
 
 		try {
 			await requestCompletion( prompt );
+			if ( ! shortTinyEditor.getContent() || shortDescriptionGenerated ) {
+				await requestShortCompletion(
+					`Please provide a brief, 1-2 sentence summary of the following product description in fewer than 50 words:\n ${ tinyEditor.getContent() }`
```

Comment:
I think we will want to test out some alternative prompts here, specifically, ones that are geared towards writing meta snippets that appear under SERPs. For instance, I got a description with this PR that was too long to fit underneath a search engine result (it should be max 155 characters):

![Screenshot 2023-07-21 at 17 10 06](https://github.com/woocommerce/woocommerce/assets/6723003/1d37aa06-fe4a-478c-95dc-d70a86d6b940)
![Screenshot 2023-07-21 at 17 13 57](https://github.com/woocommerce/woocommerce/assets/6723003/ccbd77aa-26f0-4d99-b664-c1fbd63e6aa2)

It gets cut off when it's too long which leads to awkward results.

I tried writing a longer prompt which seemed to provide decent results:
```
Please write a high-converting Meta Description for the following WooCommerce product description. It should entice someone from a search results page to click on the product link. 
It should be no more than 155 characters so that the entire meta description fits within the space provided by the search engine result without being cut off or truncated. It should explain what users will see if they click on the product page link. It should include the target keyword for the product: {description here}
```

![Screenshot 2023-07-21 at 17 20 14](https://github.com/woocommerce/woocommerce/assets/6723003/41dc5be7-17f1-425b-982f-442a944736f9)

Reply:
@tommyshellberg Nice, I like it! I actually didn't give the prompt too much thought 😆. I updated this with your prompt with just a couple minor tweaks based on what I observed.